src/Microsoft.DotNet.Cli.Utils/CsvFormatter.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.DotNet.Cli.Utils
+{
+    internal static class CsvFormatter
+    {
+        internal static string EscapeRow(IEnumerable<string> values)
+        {
+            return values
+                .Select(EscapeValue)
+                .Aggregate((a, v) => a + "," + v);
+        }
+
+        internal static string EscapeValue(string value)
+        {
+            return "\"" + value.Replace("\\", "\\\\").Replace("\"", "\\\"") + "\"";
```

Comment:
@anurse, this is the serialization mechanism that was in place for the deps file generator. If we move closer to RFC4180 we'll need a matching change in the host. However, I think you were planning to change the deps file format altogether, so perhaps this should just wait and then get removed?